add 117 Model server (#808)
openvinotoolkit/openvino_notebooks: notebooks/README.md

| [113-image-classification-quantization](113-image-classification-quantization/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?labpath=notebooks%2F113-image-classification-quantization%2F113-image-classification-quantization.ipynb) | Image Classification Models with POT |
| [114-quantization-simplified-mode](114-quantization-simplified-mode/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?labpath=notebooks%2F114-quantization-simplified-mode%2F114-quantization-simplified-mode.ipynb) | Quantize Image Classification Models with POT in Simplified Mode|
| [115-async-api](115-async-api/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?labpath=notebooks%2F115-async-api%2F115-async-api.ipynb) | Use Asynchronous Execution to Improve Data Pipelining| |
| [116-sparsity-optimization](116-sparsity-optimization/)<br> | Improve performance of sparse Transformer models |

> | [117-model-server](notebooks/117-model-server/)| A basic introduction to using OpenVINO™ Model Server (OVMS) |

Reviewer: ```suggestion
| [117-model-server](notebooks/117-model-server/)| Introduction to model serving with OpenVINO™ Model Server (OVMS) |
```
